backend/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 version: '3'
 services:
   stateoftheaeapps-backend:
-    build: .
```

Comment:
Do not remove this line, just comment it out until we setup circleCI. Also, for other branches that will not use stable image it is better to leave the option to build image upon starting docker-compose.